common/messaging/kafka/clientImpl.go

```diff
@@ -86,16 +86,20 @@ func NewKafkaClient(
 // NewConsumer is used to create a Kafka consumer
 func (c *clientImpl) NewConsumer(app, consumerName string) (messaging.Consumer, error) {
 	topics := c.config.GetTopicsForApplication(app)
+	version, err := sarama.ParseKafkaVersion(c.config.GetKafkaVersionForApplication())
+	if err != nil {
+		version = sarama.V0_10_2_0
```

Comment:
Swallowing errors is not a good idea. There could be some errors that we want to let users be aware of. 
You could do:
```go
kafkaConfig := c.config.GetKafkaVersionForApplication()
if kafkaConfig.version == ""{
     kafkaConfig.version = "0.10.2.0"
}
version, err := sarama.ParseKafkaVersion(kafkaConfig)
...
```